internal/controller/accessrequest_controller.go

```diff
@@ -55,6 +55,7 @@ const (
 	// managed by this controller
 	AccessRequestFinalizerName = "accessrequest.ephemeral-access.argoproj-labs.io/finalizer"
 	roleTemplateField          = ".spec.roleTemplateName"
+	projectField               = ".status.TargetProject"
```

Comment:
Not a huge deal, but the capitalized first letter in `TargetProject` is a bit odd.

Reply:
Not odd. It was wrong! :P 
Fixed :)